modules/platform/nuxeo-platform-task/nuxeo-platform-task-api/src/main/java/org/nuxeo/ecm/platform/task/TaskService.java

```diff
@@ -254,14 +254,16 @@ String endTask(CoreSession coreSession, NuxeoPrincipal principal, Task task, Str
      *
      * @since 5.7.3
      */
-    void reassignTask(CoreSession session, String taskId, List<String> actors, String comment);
+    void reassignTask(CoreSession session, String taskId, List<String> actors, String comment,
+            Map<String, Serializable> eventInfo);
```

Comment:
For backward compatibility you will need to keep the previous signature as well for `reassignTask` and `delegateTask`. Delegate to the new one using a `default` method that passes a `new HashMap<>()` for the new parameter.